pages/common/vboxmanage-cloud.md

```diff
@@ -0,0 +1,36 @@
+# vboxmanage-cloud
+
+> VirtualBox command-line interface for managing cloud instances and images.
+> More information: <https://www.virtualbox.org/manual/ch08.html#vboxmanage-cloud>.
+
+- List Instances:
+
+`VBoxManage cloud {{provider=name}} {{profile=name}} list instances {{--state=string}} {{--compartment-id=string}}`
```

Comment:
```suggestion
`VBoxManage cloud --provider={{provider_name}} --profile={{profile_name}} list instances --state={{running|terminated|paused}} --compartment-id={{compartment_id}}`
```

Please do it like this: placeholders should be self-describing and only stuff that can be changed by an user goes into `{{}}`.